agents/check_mk_agent.macosx

```diff
@@ -192,11 +193,42 @@ tr -d ' '` | bc
 # checks plugins with subchecks for parsing that output. Maybe reduce
 # the output size by grepping away totally useless parts
 
-echo '<<<netctr>>>';
-date +'%s'; netstat -inb | egrep -v '(^Name|lo|plip)' | grep Link | awk '{
-print $1,$7,$5,$6,"0","0","0","0","0",$10,$8,$9,"0","0",$11,"0","0"; }'
-# FIXME: send netstat -inb plain, write proper check plugins for
-# clean parsing of the output
+# Mac version of lnx_if. Needs `iproute2mac` to have been installed.
+if hash ip 2>/dev/null; then
+    echo '<<<lnx_if>>>'
+
+    interfaces=$(ip a | grep '^\S' | grep -E -v '^(awdl0|utun)' | awk -F ':' '{ print $1 }')
+    counter=0
+    echo "[start_iplink]"
+    while read -r eth; do
+        counter=$((counter+1))
+        echo -n "${counter}: "
+        ip a show "$eth"
+    done <<< "$interfaces"
+    echo "[end_iplink]"
+
+    echo '<<<lnx_if:sep(58)>>>'
+    netstat=$(netstat -inbd | grep -E "$interfaces" | grep Link)
+    # Format: Name, IBytes, IPckts, IErr, Drop, 0 (fifo), 0 (frame), 0 (compressed), 0 (multicast), Coll, Obyts, OPckts, OErrs, 0 (drop), 0 (fifo), 0 (coll), 0 (drop)
+    #      (Collisions & drops are not broken out into in/out, so I'm assuming receive for now)
```

Comment:
@ThomasKaiser : Would love input here as to what the correct place for the unlabled Colls & drops really are. Right now just put them both for Receiving.